src/services/smsOTPService.ts

```diff
@@ -0,0 +1,71 @@
+import { AxiosInstance } from 'axios';
```

Comment:
@sam-corbado Rename file to smsOtpService.ts (in line with emailOtpService.ts)

Reply:
Perhaps you looked at an older commit? It's already handled in newer commits.